src/index.js

```diff
@@ -2,7 +2,7 @@ import './style.css';
 import Scores from './modules/object.js';
 import { upload, get } from './modules/API.js';
 
-/* eslint-disable no-undef */
+/* eslint-disable-next-line no-undef */
```

Comment:
That's how we have to use it. It's a library that I added and that was how we have to use it through their documentation.

Reply:
@shayan1234554321, I understand that is how the library is used, but if you research deeper, there would be a way to use the library that integrates with `Eslint`. However, you can use it for this milestone since it is the only Eslint-disable statement used in the file. 👍